lib/container.js

```diff
@@ -283,9 +284,35 @@ function loadGherkinSteps(paths) {
   global.After = fn => event.dispatcher.on(event.test.finished, fn);
   global.Fail = fn => event.dispatcher.on(event.test.failed, fn);
 
-  for (const path of paths) {
-    loadSupportObject(path, `Step Definition from ${path}`);
+  //If gherkin.steps includes only one *.js, then this will iterate through that folder and send all step def js files to loadSupportObject
+  //This is done so that we need not enter all Step Definition files under config.gherkin.steps
+  //This works only for one level folder, not for subfolders
+  if(paths.length == 1 && paths[0] == './step_definitions/*.js'){
+    let folderPath = paths[0];
+    if (folderPath.charAt(0) === '.') {
+      folderPath = path.join(global.codecept_dir, folderPath);
+    }
+    if(folderPath != "") {
+      glob(folderPath, function(err, files){
+        if(err){
+          console.log(err);
+        }
+        else{
+          for(const file of files){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected space(s) after "for". (keyword-spacing)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4548869)